scripts/SANS/sans/algorithm_detail/move_workspaces.py

```diff
@@ -611,7 +632,7 @@ def do_move_initial(self, move_info, workspace, coordinates, component):
                                     DetectorType.to_string(DetectorType.LAB))
 
         # Shift the low-angle bank detector in the x direction
-        angle = -coordinates[0]
+        angle = coordinates[0]
```

Comment:
Yes, comparing it again with the old script showed that the the minus sign was wrong here.